Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -68,18 +83,23 @@ getCustomerMobileNumber callSid callFrom_ callTo_ callStatus = do
         >>= fromMaybeM (RiderDetailsNotFound riderId.getId)
   requestorPhone <- decrypt riderDetails.mobileNumber
   callId <- generateGUID
-  callStatusObj <- buildCallStatus activeRide.id callId callSid (exotelStatusToInterfaceStatus callStatus)
+  callStatusObj <- buildCallStatus activeRide.id callId callSid (exotelStatusToInterfaceStatus callStatus) driver.id digits
   runTransaction $ QCallStatus.create callStatusObj
+  Redis.del enteredDtmfFlowKey
   return requestorPhone
   where
     dropFirstZero = T.dropWhile (== '0')
-    buildCallStatus rideId callId exotelCallId exoStatus = do
+    buildCallStatus rideId callId exotelCallId exoStatus driverId alternateNumber = do
       now <- getCurrentTime
+      enteredDtmfFlow <- Redis.get (makeEnteredDtmfFlowKey callSid) >>= fromMaybeM (InvalidRequest "Entered-Dtmf-Flow Not Found")
       return $
         SCS.CallStatus
           { id = callId,
             callId = exotelCallId,
             rideId = rideId,
+            driverEntityId = driverId,
```

Comment:
why to driverId, alternateNumber?

Reply:
added three columns for storing the driverId , the alternate number from which he has called so since at every call we are updating the call status table we won't know which driver went through dtmf flow so added a flag -- entered dtmf flow to explicitly recognise which driver went through this flow. Ops team will call this driver and tell him to add his alternate number to his account

Reply:
We can get the driver using rideId. No need to add one extra column here.